sys-apps/systemd-utils/files/systemd-utils-254_rc3-dt_relr.patch

```diff
@@ -0,0 +1,26 @@
+https://bugs.gentoo.org/910570
```

Comment:
You should be able to drop this, it's in 254.

Reply:
Gotcha.